target/sample-collector-values.yaml

```diff
@@ -0,0 +1,100 @@
+opentelemetry-collector:
+  config:
+    exporters:
+      logging:
+        verbosity: basic
+      otlphttp/customer:
+        metrics_endpoint: ${CUSTOMER_METRICS_ENDPOINT}
+        traces_endpoint: ${CUSTOMER_TRACES_ENDPOINT}
+      otlphttp/kfendpoint:
+        metrics_endpoint: ${KF_METRICS_ENDPOINT}
```

Comment:
KF_*_ENDPOINT is basically observe, right? Then why not hard code it?
Also, for KF_METRICS_ENDPOINT - can be ensure we get the org_id label properly?

Reply:
I don't think this is correct.

We don't want target otel to send directly to observe. As discussed offline. This should be similar to how we collect using analytics-service.

otel col -> customers kfuse -  (analytics-service  in customer's kfuse install will then periodically query/create metrics) -> scraped by kfuse-observability-agent -> kfuse